cuML/src/solver/solver.cu

```diff
@@ -340,5 +338,158 @@ void sgdPredictBinaryClass(const double *input, int n_rows, int n_cols,
 
 }
 
+void cdFit(float *input,
```

Comment:
This appears to be our 'interface' method. So, exposing `cumlHandle` is the way to go. The PR's I mentioned previously should give enough guidelines for doing so.